src/core/config/remote/remote_config_data_wrapper.ts

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the ""License"");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an ""AS IS"" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { SoloError } from '../../errors.js'
+import * as version from '../../../../version.js'
+import * as yaml from 'yaml'
+import { RemoteConfigMetadata } from './metadata.js'
+import { ComponentsDataWrapper } from './components_data_wrapper.js'
+import * as constants from '../../constants.js'
+import type { Cluster, Version, Namespace, RemoteConfigData, RemoteConfigDataStructure } from './types.js'
+import type * as k8s from '@kubernetes/client-node'
+import type { ToObject, Validate } from '../../../types/index.js'
+
+export class RemoteConfigDataWrapper implements Validate, ToObject<RemoteConfigDataStructure> {
+  private readonly _version: Version = version.HEDERA_PLATFORM_VERSION
```

Comment:
```suggestion
  private readonly _version: Version = '1.0.0'
```
the comment in the [notion page](https://www.notion.so/swirldslabs/Multi-user-and-Multi-cluster-Support-Solo-State-and-Lease-Mechanism-DRAFT-8b013e8befe04e3daffc889619c851c1?pvs=4) was
`# solo state maintaines its own version increments, not same as solo version`